integration-tests/src/test/java/org/openmrs/maven/plugins/SetupIntegrationTest.java

```diff
@@ -215,6 +215,26 @@ public void setup_shouldInstallServerWithGivenJavaHomeAndAddJavaHomeToSdkPropert
         assertThat(server, serverHasDebugPort("1044"));
     }
 
+    @Test
+    public void setup_shouldInstallWithParentDistroSpecifiedInDistroProperties() throws Exception{
```

Comment:
Could you please add a test for doing it recursively? e.g setup_shouldInstallWithParentAndGrandparentDistro